### PR TITLE
 Execute `dotnet-nswag.dll` with `--roll-forward-on-no-candidate-fx 2`

### PR DESCRIPTION
**Pull request by dougbu**
- #2195

nits:
- use new name of `swagger2csclient` command
- ignore _all_ .nupkg files

### CODE REVIEW

**Comment by dougbu** [comment]
@RicoSuter I realize #2195 is closed but this at least addresses part of the 3.0 compat issue

File: src/NSwag.ApiDescription.Client/NSwag.ApiDescription.Client.targets [hunk 1]
```diff
@@ -2,7 +2,8 @@
 <Project>
   <PropertyGroup>
     <_NSwagCommand>$(NSwagExe)</_NSwagCommand>
-    <_NSwagCommand Condition="'$(MSBuildRuntimeType)' == 'Core'">$(NSwagExe_Core21)</_NSwagCommand>
+    <_NSwagCommand
+        Condition="'$(MSBuildRuntimeType)' == 'Core'">dotnet --roll-forward-on-no-candidate-fx 2 $(NSwagDir_Core21)/dotnet-nswag.dll</_NSwagCommand>
```
Comment (RicoSuter) [inline src/NSwag.ApiDescription.Client/NSwag.ApiDescription.Client.targets]: Why arent we checking other fw versions like net core 2.2?

**Comment by RicoSuter** [comment]
So this should be enough for now?

File: src/NSwag.ApiDescription.Client/NSwag.ApiDescription.Client.targets (hunk 1 above)
Comment (dougbu) [reply src/NSwag.ApiDescription.Client/NSwag.ApiDescription.Client.targets]: This code uses the MSBuild runtime environment to guess whether _any_ version of `dotnet` is installed. Figuring out the version when the tool command of interest works with 2.1 or later installed is extra work for minimal benefit.

**Comment by dougbu** [comment]
> So this should be enough for now?

Yes, it works in a 3.0-only environment just fine.

**Comment by RicoSuter** [comment]
Ok then - v13 will be released in the next days... need to update the wiki, do more tests etc. 
Is there documentation about how to use this service reference stuff, the supported scenarios, etc.? I'd like to verify whether all important things are covered, if something is missing etc...

**Comment by RicoSuter** [comment]
Thanks for the PR!

**Comment by dougbu** [comment]
> Is there documentation about how to use this service reference stuff, the supported scenarios, etc.? I'd like to verify whether all important things are covered, if something is missing etc...

Not yet though it's definitely due. This lack exists in part because preview 6 isn't released yet and those bits contain the Web project side of things. The following is available:
- https://github.com/aspnet/AspNetCore/blob/master/src/Mvc/Extensions.ApiDescription.Client/src/build/Microsoft.Extensions.ApiDescription.Client.props contains some docs for the user-visible properties and items usable in "client" projects
- https://github.com/aspnet/AspNetCore/blob/master/src/Mvc/Extensions.ApiDescription.Server/src/build/Microsoft.Extensions.ApiDescription.Server.props contains some docs for the user-visible properties usable in "server" (Web) projects
  - These features will work end-to-end after we release 3.0 preview 6 and you publish your next release after my next PR (coming in the next few days)

**Pull request merged**